recipes/dcase2022_task4_baseline/README.md

```diff
@@ -19,6 +19,14 @@ Run the command `python generate_dcase_task4_2022.py --basedir="../../data"` to
 
 The dataset uses [FUSS][fuss_git], [FSD50K][FSD50K], [desed_soundbank][desed] and [desed_real][desed].
 
+**Common issues:**
+
+`FileNotFoundError: [Errno 2] No such file or directory: 'ffprobe'`
+
+it probably means you have to install ffmpeg on your machine.
+
+you can solve it using `sudo apt install ffmpeg`
```

Comment:
```suggestion
It probably means you have to install `ffmpeg` on your machine. 
A possible installation: `sudo apt install ffmpeg`
```

Reply:
Agree with the comment from Nicolas, plus, this file is causing a conflict. In fact, the new read me file is a bit different. 
Moreover, this part is out of context here. In his particular part of the READ me file we are describing the dataset and the different data used in the set. I think this suggestion should go under the section **Requirements**.